spritz/README.md

```diff
@@ -0,0 +1,153 @@
+animation if the user stops dragging forwards at any point (before starting the autoplay animation, if any). If you don't set this, it will default to
+`LinearInterpolator`. You can always override this default for single pages (see [`withSwipeForwardInterpolator`](#withswipeforwardinterpolator)).
+
+#### `withDefaultSwipeBackwardsInterpolator`
+
+`withDefaultSwipeBackwardsInterpolator(TimeInterpolator swipeBackwardsInterpolator)` sets the interpolator to use for all pages to complete the swipe 
+animation if the user stops  dragging backwards at any point. If you don't set this, it will default to `LinearInterpolator`. You can always override 
+this default for single pages (see [`withSwipeBackwardsInterpolator`](#withswipebackwardsinterpolator)).
+
+#### `withPages`
+
+`withPages(SpritzPage... pages)` lets you specify the number of pages to animate through. The library is made to work with the same number of pages as 
+the ones set in the `ViewPager`.
+
+See [Building the pages](#building-the-pages) to learn how to build a list of pages that match your exported animation.
+
+#### `attachTo`
+
+After setting the pages, simply call `attachTo(ViewPager viewPager)` to register all the needed listeners and set everything up. Congratulations! Now
+you have a `spritz` object!
+
+#### `startPendingAnimations`
+
+You can call `startPendingAnimations` on your `spritz` object to trigger the autoplay animation for your first page, if you have one.
+
+### Building the pages
+
+To build a page, use the `SpritzPage.Builder` class, configure it as explained in the following sections, then build it with `.build()`.
+
+#### `withAutoPlayDuration`
+
+`withAutoPlayDuration(long duration, TimeUnit timeUnit)` sets the duration of the autoplay animation for the given page. This must match the exact 
+duration of the autoplay segment in your AfterEffects project.
+
+#### `withSwipeDuration`
+
+`withSwipeDuration(long duration, TimeUnit timeUnit)` sets the duration of the swipe animation for the given page. This must match the exact 
+duration of the swipe segment in your AfterEffects project.
+
+#### `withSwipeForwardInterpolator`
+
+`withSwipeForwardInterpolator(TimeInterpolator timeInterpolator)` sets the interpolator to use to complete the swipe animation if the user stops 
+dragging forwards at any point (before starting the autoplay animation, if any). If you don't set this, it will default to the one you set in 
+`withDefaultSwipeForwardInterpolator` or a `LinearInterpolator` if you didn't set one.
+
+#### `withSwipeBackwardsInterpolator`
+
+`withSwipeBackwardsInterpolator(TimeInterpolator timeInterpolator)` sets the interpolator to use to complete the swipe animation if the user stops 
+dragging backwards at any point. If you don't set this, it will default to the one you set in `withDefaultSwipeBackwardsInterpolator` or a 
+`LinearInterpolator` if you didn't set one.
+
+## License
```

Comment:
What about a `LICENSE` file?

Reply:
Done in bd30c37c43e4749853fb732b993c05c7cc01a824.